docs/source/release_notes/release-notes.rst

```diff
@@ -42,6 +42,10 @@ develop
          - Change
 
     *    - |improved|
+         - Add support for generating schemas with Java8 Optionals instead of Guava Optionals. Note that this change
```

Comment:
currently, this is a breaking change, as all products will have to change all of their schemas (even historical ones). Fixing the constructors will fix this (and also mean not having to do so many changes to constructors).